.github/workflows/ci.yml

```diff
@@ -74,7 +74,30 @@ jobs:
     - name: Wait for chart to get published
       run: sleep 120
     - name: Update component version
-      uses: SwissDataScienceCenter/renku/actions/update-component-version@0.7.11
+      uses: SwissDataScienceCenter/renku/actions/update-component-version@master
       env:
         CHART_NAME: renku-notebooks
         GITHUB_TOKEN: ${{ secrets.DEPLOY_TOKEN }}
+
+  publish-chart-latest:
```

Comment:
I'd rename this to "push-latest-image" as we're not publishing a "latest" chart, just pushing images.